core/src/main/java/org/opensearch/sql/calcite/udf/datetimeUDF/TimestampFunction.java

```diff
@@ -20,7 +19,8 @@
 /**
  * We need to write our own since we are actually implement timestamp add here
  * (STRING/DATE/TIME/DATETIME/TIMESTAMP) -> TIMESTAMP (STRING/DATE/TIME/DATETIME/TIMESTAMP,
- * STRING/DATE/TIME/DATETIME/TIMESTAMP) -> TIMESTAMP
+ * STRING/DATE/TIME/DATETIME/TIMESTAMP) -> TIMESTAMP It's implicitly transferred into timestamp, so
```

Comment:
please format this java doc for more readable

Reply:
Done.